src/lib/coda_base/transaction_logic.ml

```diff
@@ -1516,11 +1512,35 @@ module Make (L : Ledger_intf) : S with type ledger := L.t = struct
         ~modify_timing:(fun acc ->
           update_timing_when_no_deduction ~txn_global_slot acc )
     in
+    let compute_balance account_id =
+      match get_user_account_with_location t account_id with
+      | Ok (`Existing _, account) ->
+          Some account.balance
+      | _ ->
+          None
+    in
+    let balances =
+      match Fee_transfer.to_singles transfer with
+      | `One ft ->
+          { User_command_status.Balance_data.fee_payer_balance=
```

Comment:
Change `User_command_status` to `Transaction_status`?

Reply:
Punting this as a TODO for a follow-up PR. We want to land #7087 that depends on this ASAP, and the inappropriateness of this name hasn't been changed in this PR.